.swiftlint.yml

```diff
@@ -1,10 +1,90 @@
-# inherit from Kodeco Swift Style Guide - https://github.com/kodecocodes/swift-style-guide
-parent_config: https://raw.githubusercontent.com/kodecocodes/swift-style-guide/345b5690a99bb6212fd9bbf17f673a8a0d7a5607/com.raywenderlich.swiftlint.yml
+excluded:
+  - **/*+Generated.swift
+  - **/Ampli.swift
 
-# we disable any particularly pedantic rules which do not support auto-correction
 disabled_rules:
+  - discarded_notification_center_observer
+  - notification_center_detachment
+  - orphaned_doc_comment
+  - todo
+  - unused_capture_list
   - attributes
   - indentation_width
+  - function_parameter_count
+
+analyzer_rules:
+  - unused_import
+
+opt_in_rules:
+  - array_init
+  - attributes
+  - closure_end_indentation
+  - closure_spacing
+  - collection_alignment
+  - colon
+  - convenience_type
+  - discouraged_object_literal
+  - empty_collection_literal
+  - empty_count
+  - empty_string
+  - enum_case_associated_values_count
+  - fatal_error_message
+  - first_where
+  - force_unwrapping
```

Comment:
Wondering what reason we have for including `force_unwrapping`? I think there's various cases where this should certainly be allowed and imo doesn't really require lint warnings

Reply:
@jvanderwee Im up for removing this, there's loads of legit cases in consumer where we force unwrap

Reply:
it is allowed (albeit with a warning), IMO should be avoided